### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by JeremyButts**
Refer to issue [54](https://github.com/splunk/splunk-sdk-csharp-pcl/issues/54).

### CODE REVIEW

**Comment by JeremyButts** [comment]
Any questions, comments or concerns regarding this pull request?

**Comment by shakeelmohamed** [comment]
@JeremyButts Sorry about the major delay in getting back to you!
We're looking at bringing this in for the next release, can you remove the NCrunch file? (`src/Splunk.Client/nCrunchTemp_927b2bcf-1091-4b97-a65b-a51cfe2ec2ff`)

File: test/unit-tests/TestSearchResultStream.cs [hunk 1]
```diff
@@ -14,6 +14,8 @@
  * under the License.
  */
 
+using System.Linq;
```
Comment (shakeelmohamed) [inline test/unit-tests/TestSearchResultStream.cs]: Is this being used? If so, can we move it into the `namespace` with the others?

**Comment by shakeelmohamed** [comment]
@JeremyButts we've released v2.2.5 which addresses this issue. Please give it a try

**Pull request closed without merging**